### DISCUSSION TITLE
php artisan [command] error

### DISCUSSION THREAD

**Discussion by igorsamardzija** [wintercms/winter#256] [Q&A] [answered]
Hi Guys,

New in Winter. 
Used October v.1 before. 

Trying to install via Composer but getting error 
**Translator.php line 304:      
  Undefined index: en** 

Basically when I run any php artisan [command] I`m getting same error

System
Ubuntu 20.04
PHP 7.4.3
Composer version 2.1.5

Any solution on this?

**Comment by LukeTowers** [comment, marked as answer]
[EDIT]: October's fix doesn't actually solve the problem, just hides it. See https://github.com/briannesbitt/Carbon/pull/2398 for the actual fix which should hopefully be merged into Carbon and released shortly.

[Original]:
Sounds like this just recently popped up for October as well, could you test making the change from https://github.com/octobercms/library/commit/ff2cec61078bcfc3c75117a37f8ea137458ebf91 and let me know how that goes for you? If it works we can implement a similar fix for everyone on Winter.

**Reply by arvislacis** [reply]
Can confirm that applying mentioned October CMS patch works and now `composer update` works without problems. :)

**Reply by LukeTowers** [reply]
Sounds good, I messaged you in Discord so I can attempt to replicate locally

**Comment by igorsamardzija** [comment]
This fixed my issue 
https://github.com/briannesbitt/Carbon/pull/2398/commits/6b5f440d44b87f9c20c0e1656c5959838d633535

Thanks for help
